src/Common/tests/System/PlatformDetection.cs

```diff
@@ -41,6 +41,10 @@ public static partial class PlatformDetection
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 16215;
         public static bool IsArmProcess => RuntimeInformation.ProcessArchitecture == Architecture.Arm;
         public static bool IsNotArmProcess => !IsArmProcess;
+        // Windows 10 Insider Preview Build 16215 introduced the necessary APIs for the UAP version of
```

Comment:
I think there is some words in the CoreFx style guideline about this.  In general, I agree that "comment sandwiches" are not something we usually do.